public/css/grocy_night_mode.css

```diff
@@ -312,3 +312,9 @@ body.night-mode {
 .night-mode .text-muted {
 	color: #8f9ba5 !important;
 }
+
+.night-mode .qr-code {
+	background-color: #eee;
+	padding: 10px;
+	border-radius: 10px;
+}
```

Comment:
I was just going to open a PR for the QR code in night mode as well.
The quiet zone (padding) should be 32 pixels, though, as the standard mandates 4 modules of quiet zone. The QR code is generated with a scale factor of 4 which results in a module size of 8 pixels.

To be more robust against future changes, it might make sense to override the background color in the day mode CSS as well.

Reply:
> The quiet zone (padding) should be 32 pixels, though, as the standard mandates 4 modules of quiet zone. The QR code is generated with a scale factor of 4 which results in a module size of 8 pixels.

Which standard are you talking about? I was able to scan fine personally.
> To be more robust against future changes, it might make sense to override the background color in the day mode CSS as well.

Good point. However that isn't really "Night mode improvements"... Feel free to make a PR once this gets merged.

Reply:
> Which standard are you talking about?

The standard itself is non-free, but see https://en.wikipedia.org/wiki/QR_code#/media/File:QRCode-2-Structure.png or https://www.qrcode.com/en/howto/code.html for an explanation.

>  I was able to scan fine personally.

Yes, most scanners don't need that much quiet zone around the code, but I would not depend on that.

> Good point. However that isn't really "Night mode improvements"... Feel free to make a PR once this gets merged.

Yes, absolutely. That's out of scope for this PR. Was just a thing that came to mind.

Reply:
I updated that. If you think it's good enough, then you can resolve the conversation.

Reply:
I have to admit I don't understand the purpose of the `1.5vw` value as that scales with viewport size while the QR code size itself is fixed. For usual screen sizes it's fine as the `20px` value takes over.

Reply:
It's *just in case* if you're using it on a phone, or tablet.